code/controllers/subsystem/machinery.dm

```diff
@@ -11,11 +11,14 @@ var/list/machines = list()
 	display_order = SS_DISPLAY_MACHINERY
 
 	var/list/currentrun
+	var/currentrun_index
 
+	var/obj/machinery/M //Machine currently being processed.
+	var/c
```

Comment:
Accessing a datum var is slower than accessing a local var.

Reply:
I didn't know that, ty.